web-platform/website/src/components/pages/profile/ProfileButtons.tsx

```diff
@@ -26,17 +26,21 @@ export function ProfileButtons() {
     <Row>
       <Col className="d-flex justify-content-center">
         <Button
-          className={`d-flex fw-bold ps-4 pe-4 pt-2 pb-2 ${
-            !isValid ? "border-danger text-danger" : SecondaryButton
-          } me-3`}
+          className={`d-flex fw-bold ps-4 pe-4 pt-2 pb-2 ${!isValid ? "border-danger text-danger" : SecondaryButton
+            } me-3`}
           outline={!isValid}
           disabled={!isValid}
           onClick={() => {
             updateUserDocument({
               demographicsData: profileData,
               onboared: true,
             });
-            goToStudiesPage();
+            if (window) {
```

Comment:
I believe this is reverted and we will have a function call in the end commit?

Reply:
Yes, I removed it. Sorry this was a messy PR. Sigh